core/src/main/java/org/jdbi/v3/core/statement/ParsedSql.java

```diff
@@ -72,6 +72,26 @@ public String toString() {
             + '}';
     }
 
+    /**
+     * A static factory of {@link ParsedSql} instances. The first
+     * argument, i.e. SQL, must contain only positional parameters
```

Comment:
More to the point, `?` is a magic token in JDBC.

Reply:
I understand this is just a note. Not something I should describe in javadoc?